skins/material3/QskMaterial3Skin.cpp

```diff
@@ -151,10 +151,10 @@ namespace
         const QskMaterial3Theme& m_pal;
     };
 
-    QFont createFont( int pixelSize, qreal tracking, QFont::Weight weight )
+    QFont createFont( const QString& name, int lineHeight, int size, qreal tracking, QFont::Weight weight )
     {
-        QFont font( "Roboto" );
-        font.setPixelSize( pixelSize );
+        QFont font( name, size );
```

Comment:
not sure if we need to set the point size, since we set the pixel size later anyhow. However, for some reason Material always gives both metrics, see e.g. https://m3.material.io/styles/typography/type-scale-tokens#d74b73c2-ac5d-43c5-93b3-088a2f67723d